src/es/components/atoms/arrow/Arrow.js

```diff
@@ -18,34 +18,23 @@ import { Shadow } from '../../prototypes/Shadow.js'
  *  --color-hover [white]
  * }
  */
-export default class Arrow extends Shadow() {
+export default class Arrow extends Hover() {
   static get observedAttributes () {
     return ['hover']
   }
 
-  constructor (...args) {
+  constructor (options = {}, ...args) {
     super(...args)
-
-    this.mouseoverListener = event => {
-      if (this.hasAttribute('move') && !this.hasAttribute('hover-set-by-outside')) this.setAttribute('hover', 'true')
-      this.classList.add('hover')
-    }
-    this.mouseoutListener = event => {
-      if (this.hasAttribute('move') && !this.hasAttribute('hover-set-by-outside')) this.setAttribute('hover', '')
-      this.classList.remove('hover')
-    }
   }
 
   connectedCallback () {
+    super.connectedCallback()
     if (this.shouldRenderCSS()) this.renderCSS()
     if (this.shouldRenderHTML()) this.renderHTML()
-    this.mouseEventElement.addEventListener('mouseover', this.mouseoverListener)
-    this.mouseEventElement.addEventListener('mouseout', this.mouseoutListener)
   }
 
   disconnectedCallback () {
-    this.mouseEventElement.removeEventListener('mouseover', this.mouseoverListener)
-    this.mouseEventElement.removeEventListener('mouseout', this.mouseoutListener)
+    super.disconnectedCallback()
     this.parentNodeShadowRootHost = null
```

Comment:
`this.parentNodeShadowRootHost = null` kannst du guten gewissens entfernen und dann ist ja der disconnectedCallback leer. somit kannst du den ganzen disconnectedCallback entfernen.